intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -1109,6 +1109,10 @@ \subsection{Generic resource requirements}
 If you omit this option, the queue manager will not complain but use a
 default value (one hour on most clusters).
 
+\ifgent
+The highest maximum value for walltime is \strong{72 hours}.
```

Comment:
"highest maximum" -> "maximum"

Maybe clarify this is specific to HPC-UGent clusters (other sites have different maxima)